apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -90,6 +92,65 @@ export const siteRouter = router({
       })
       return getSiteConfig(db, id)
     }),
+  updateSiteConfig: protectedProcedure
+    .input(updateSiteConfigSchema)
+    .mutation(async ({ ctx, input: { siteId, siteName } }) => {
+      await validateUserPermissionsForSite({
+        siteId,
+        userId: ctx.user.id,
+        action: "update",
+      })
+      const {
+        user: { id: userId },
+      } = ctx
```

Comment:
oh curious, why this ah? cos we have been just calling `ctx.user.id` in other part of the codebase. just thought to be consistent unless there's a reason